src/renderer/routes/test_sequencer_panel/components/DesignBar.tsx

```diff
@@ -48,24 +55,30 @@ export function DesignBar() {
   const importSequences = useImportSequences();
 
   const [summary, setSummary] = useState<Summary>({
-    numberOfTestRun: 0,
-    numberOfTest: 0,
+    numberOfTestRunInSeq: 0,
+    numberOfTestInSeq: 0,
+    numberOfTestRunInTotal: 0,
+    numberOfTestInTotal: 0,
     numberOfSequenceRun: 0,
     numberOfSequence: 0,
     successRate: 0,
     status: "pending",
   });
   useMemo(() => {
     setSummary({
-      numberOfTestRun: getNumberOfTestRun(elems),
-      numberOfTest: getNumberOfTest(elems),
+      numberOfTestRunInSeq: getNumberOfTestRunInSeq(elems),
+      numberOfTestInSeq: getNumberOfTestInSeq(elems),
+      numberOfTestRunInTotal: getNumberOfTestRunInTotal(sequences),
+      numberOfTestInTotal: getNumberOfTestInTotal(sequences),
```

Comment:
does this not work? `useState` seem unnecessary
```typescript
  const {
    numberOfTestRun,
    numberOfTest,
    numberOfTestRunInSeq,
    numberOfTestInSeq,
    numberOfTestRunInTotal,
    numberOfTestInTotal,
    numberOfSequenceRun,
    numberOfSequence,
    successRate,
    status,
  } = useMemo(() => {
    return {
      numberOfTestRun: getNumberOfTestRun(elems),
      numberOfTest: getNumberOfTest(elems),
      numberOfTestRunInSeq: getNumberOfTestRunInSeq(elems),
      numberOfTestInSeq: getNumberOfTestInSeq(elems),
      numberOfTestRunInTotal: getNumberOfTestRunInTotal(sequences),
      numberOfTestInTotal: getNumberOfTestInTotal(sequences),
      numberOfSequence: getNumberOfSequence(sequences),
      numberOfSequenceRun: getNumberOfSequenceRun(sequences),
      successRate: getSuccessRate(elems),
      status: getGlobalStatus(cycleRuns, sequences, elems),
    };
  }, [elems, sequences, cycleRuns]);
  ```

Reply:
Yeah, my implementation was dumb